test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -142,23 +142,18 @@ protected long LastRaftIndexForCommand(RavenServer server, string commandType)
             return updateIndex;
         }
 
-        protected long CountOfRaftCommandByType(RavenServer server, string commandType)
+        protected IEnumerable<DynamicJsonValue> GetRaftCommandByType(RavenServer server, string commandType = null)
```

Comment:
Given that the parameter `commandType` is optional I would rather changed the method name to `GetRaftCommand`.